src/ukify/ukify.py

```diff
@@ -1587,6 +1708,14 @@ def __call__(
         config_key='UKI/DeviceTreeAuto',
         config_push=ConfigItem.config_list_prepend,
     ),
+    ConfigItem(
+        '--hwids-directory',
+        metavar='PATH',
+        type=Path,
+        action='append',
```

Comment:
Hmm, so the directory can appear more than once? I don't think would work. Omit `action`.